tree/NetworkFetcher/Model/Articles.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Articles.swift
+//  tree
+//
+//  Created by ParkSungJoon on 23/01/2019.
+//  Copyright © 2019 gardener. All rights reserved.
+//
+
+import Foundation
+
+public struct Articles: Codable {
+    public let articles: Results
+}
+
+public struct Results: Codable {
+    public let page: Int
+    public let totalResults: Int
+    public let pages: Int
+    public let results: [Article]
+}
+
+public struct Article: Codable {
+    public let uri: String
+    public let lang: String
+    public let date: String
+    public let time: String
+    public let sim: Double
+    public let url: String
+    public let title: String
+    public let body: String
+    public let source: Source
+    public let author: [Author]?
```

Comment:
배열 타입의 변수 이름은 복수형으로 써줘서 단수형 변수와 구분하는 것이 좋습니다.

```suggestion
    public let authors: [Author]?
```